gateways/nmi/request_builders.go

```diff
@@ -2,8 +2,10 @@ package nmi
 
 import (
 	"fmt"
-	"github.com/BoltApp/sleet"
 	"strconv"
+
+	"github.com/BoltApp/sleet"
+	"github.com/shopspring/decimal"
```

Comment:
There are a few decimal packages with no clear indication of which one is the best. This one as a github projects seems fairly healthy, it also comes with a division function which is ideal.
The other one I checked was https://www.cockroachlabs.com/blog/apd-arbitrary-precision-decimal-package/
I might switch this to use that one instead.